pkgs/development/compilers/elm/packages/patches/elmi-to-json.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/package.yaml b/package.yaml
+index ba7e34a..25f21ea 100644
+--- a/package.yaml
++++ b/package.yaml
+@@ -32,16 +32,16 @@ library:
+   - src
+   - types-from-compiler
+   dependencies:
+-  - aeson == 1.2.4.0
+-  - async == 2.1.1.1
+-  - binary == 0.8.5.1
+-  - bytestring == 0.10.8.2
+-  - containers == 0.5.10.2
+-  - directory == 1.3.0.2
+-  - filepath == 1.4.1.2
+-  - optparse-applicative == 0.14.2.0
+-  - safe-exceptions == 0.1.7.0
+-  - text == 1.2.3.0
++  - aeson
++  - async
++  - binary
++  - bytestring
++  - containers
++  - directory
++  - filepath
++  - optparse-applicative
++  - safe-exceptions
++  - text
+   ghc-options:
+   - -Wall
+   - -Werror
```

Comment:
PR in upstream is opened for some time but still without a response from the maintainer. https://github.com/stoeffel/elmi-to-json/pull/28

Reply:
you can just use `doJailbreak elmi-to-json` combinator to avoid patching.

Reply:
I have an issue with `doJailbreak`. Apparently because of elmi-to-nix uses `packages.yaml` and doesn't contain `.cabal` file. I dug into `haskell.lib` using repl but it seems to me there is not build in a way to deal with it so I think patch might be still the best option.

```
jailbreak-cabal: dieVerbatim: user error (jailbreak-cabal: Error Parsing: file "elmi-to-json.cabal" doesn't exist.
```

Reply:
That's a bug in nixpkgs then, jailbreak should come after hpack has been ran.

Reply:
https://github.com/NixOS/cabal2nix/issues/421

Reply:
this was merged in upstream so I think we now ca avoid both `doJailbreak` and patch. https://github.com/stoeffel/elmi-to-json/pull/28

Reply:
@domenkozar elmi-to-json updated. I think this is all so this is now ready to be review again.